### DISCUSSION TITLE
The animation can trigger migraines

### DISCUSSION THREAD

**Discussion by MarjaE2** [community/community#17009] [Accessibility] [answered]
If I visit the front page here:

https://github.com/

I get hit with a lot of animation, and can get a migraine as a result. For example, if I click on the search bar on the upper left, the menu text slides to the right. This hurts, and triggers my migraines.

If I scroll down, the left side of the screen, following standard web design practices, soon stops scrolling, while the rest of the page continues. This hurts, and triggers my migraines.

**Comment by MarjaE2** [comment]
Similar problems here:

https://docs.github.com/en/get-started/quickstart/hello-world

**Comment by MarjaE2** [comment]
And here:

https://docs.github.com/en/issues

**Comment by benjiallen** [comment]
@MarjaE2 do you make use of "[prefers reduce motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences)" at the operating system level? I'm wondering if we should do more for users with this setting and whether this technique could be used to help you in this situation.

**Comment by MarjaE2** [comment]
Yes.

**Reply by benjiallen** [reply]
ok, thanks for sharing. Let me take this feedback away and see what we can do.

**Comment by cehfisher** [comment, marked as answer]
Hello @MarjaE2 - thank you for being a part of our community and taking the time to submit feedback. I'm happy to report that a pause/play button has been added to the globe animation for the logged-out GitHub homepage experience.

I am including two screenshots, the first image shows the default state with a red arrow pointing to the "pause" button. The second image shows the toggled state with a red arrow pointing to the "play" button. You will see this button when your OS `prefers-reduced-motion` option is NOT set to true/toggled on.

If your OS `prefers-reduced-motion` option IS set to true/toggled on, you will notice that all of the animations have stopped, minus the work still being done on issue [#17230](https://github.com/orgs/community/discussions/17230). We hope to have an update for that issue soon.

Please let us know if there are any additional changes you would like to see. Thanks!

Image 1:
<img width="1043" alt="Screenshot that shows the default state with a red arrow pointing to the pause button." src="https://github.com/user-attachments/assets/c18eeec1-1b07-480f-a27b-769d15c6c3bf">

Image 2:
<img width="1043" alt="Screenshot that shows the toggled state with a red arrow pointing to the play button." src="https://github.com/user-attachments/assets/c4a6d16f-6d75-4b62-9373-0ab62e1e3ad8">
